### PR TITLE
Regularly update GeoIP db

### PR DESCRIPTION
**Pull request by ozh**
Workflow to update geoip db every monday if a new version is available

### CODE REVIEW

File: .github/workflows/update-geoip.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  workflow_dispatch:
+
+# Cancels all previous workflow runs for the same branch that have not yet completed.
+concurrency:
+  # The concurrency group contains the workflow name and the branch name.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  certificate-check:
```
Comment (LeoColomb) [inline .github/workflows/update-geoip.yml:25]: It could be easier to maintain to have the two update jobs in a single workflow file

**Review by LeoColomb — CHANGES_REQUESTED** [review]
Sorry meant to do that instead

File: .github/workflows/update-geoip.yml (hunk 1 above)
Comment (ozh) [reply .github/workflows/update-geoip.yml:25]: Once everything works sure :)

File: .github/workflows/update-geoip.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+          body: |
+            Updated GeoIP database, last verified on ${{ steps.get-date.outputs.DATE }}.
+
+            Source: https://www.maxmind.com/en/account/login
+          labels: |
+            dependencies
+          reviewers: |
+            ozh
+            LeoColomb
+            dgw
```
[the remark this answers is not in the document]
Comment (ozh) [reply .github/workflows/update-geoip.yml:71]: I tried I realized I wrote it wrong :) see https://github.com/YOURLS/YOURLS/pull/3282/commits/7870c869357087c3e2c37c6844451d597405eb15#diff-ada20e574a1b16dfa05d0c19f046770d59b741fe25aeab224ef07982d9e329aaL78

**Comment by ozh** [comment]
Hmm, there's something but we're not there yet : `git clone` changes local filetime to the time of checkout, so we'll always have fresher local files than the remote ones 😅 Will improve this part

File: .github/workflows/update-geoip.yml (hunk 1 above)
Comment (ozh) [reply .github/workflows/update-geoip.yml:25]: On second thoughts I'm not so sure about this.
* The 2 workflows work differently and I think I'll have an easier time debugging when one breaks
* The cacert workflow is "safe" : before committing there's a checksum to make sure the file is valid. There is no such thing for the geoip workflow and in the possibility that the download may be corrupted we need to run unit tests on these PR

File: .github/workflows/update-geoip.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (ozh) [reply .github/workflows/update-geoip.yml]: Using team-reviewer or reviewers doesn't work as expected alas

**Pull request merged**